docs/sources/whatsnew/whats-new-in-v10-1.md

```diff
@@ -0,0 +1,498 @@
+
+### Panel plugins in Explore
+
+<!-- Ben Donnelly -->
+
+_Experimental in all editions of Grafana_
+
+Data source plugin developers can now use any plugin to visualize data in Explore. Similar to `preferredVisualizationType`, we've introduced an experimental API to render visualizations by plugin ID. In the returned data frame, set the meta option `preferredVisualisationPluginId` to the plugin ID you want to be used when showing the data for given data frame.
+
+## Alerting
+
+_All Alerting features are generally available in all editions of Grafana_
+
+We’ve made a number of improvements to simplify the alert rule creation process as well as improvements to contact points and alert management. For all the details, refer to our [Alerting documentation]({{< relref "../alerting" >}}).
+
+### Alert rules
+
+We’ve made the following changes to alert rules.
+
+#### Alert instance routing preview
+
+_This feature is for Grafana-managed alert rules only._
+
+Preview how your alert instances will be routed if they fire while you're creating your alert rule. View routing for each Alertmanager you have configured to receive Grafana-managed alerts and if required, you can easily make adjustments to your custom labels to change the way your alert instances are routed.
+
+{{< figure src="/media/docs/alerting/alert-routing-preview.png" max-width="750px" caption="Alert instance routing preview" >}}
+
+#### Alert rule types
+
+Enables you to switch to a data source-managed alert rule if your data source is configured to support alert rule creation (Ruler API enabled). By default, the alert rule type is Grafana-managed.
+
+{{< figure src="/media/docs/alerting/alert-rule-types.png" max-width="750px" caption="Alert rule types" >}}
+
+#### UI improvements
+
+- **Alert evaluation behavior**: New UI components for creating a folder and adding an evaluation group along with improved text and validation.
+- **Alert Rule list page**: The process of creating recording rules (**More** drop-down) is now separate from Grafana-managed and data source-managed alert rules (**+New alert rule**)
+  .
+- **Annotations display**: Adding a summary, description, and runbook URL as annotations are now optional. The dashboard and panel names are now also linked directly, making it easier to access.
+- **View YAML button**: Displays alert rule configuration in YAML format on the Grafana-managed alert rules form, as well as the Grafana-managed provisioned and non-provisioned Alert Rule detail view.
+- **Queries and expressions**: Several improvements have been made to the display of queries and expressions, including making **Add expression** a drop-down and moving **Conditions** to the header.
+- **Min interval option**: Improves control over query costs and performance by allowing you to adjust the minimum resolution of the data used in your alerting queries.
```

Comment:
Not sure if alerting should be capitalized here.
```suggestion
- **Min interval option**: Improves control over query costs and performance by allowing you to adjust the minimum resolution of the data used in your alerting queries.
```

Reply:
nope it can stay lower as it's not referring to the product specifically but generically to alerting functionality